packages/apps/docs/src/components/Analytics/Analytics.tsx

```diff
@@ -1,7 +1,7 @@
 import { analyticsPageView } from '@/utils/analytics';
 import { useRouter } from 'next/router';
 import Script from 'next/script';
-import React, { FC, useEffect } from 'react';
+import React, { type FC, useEffect } from 'react';
```

Comment:
I think there's a rule to separate `type` imports from "normal" imports. I'd like it to be separate and not inlined. Not sure what others think. 

Especially when you get things multiple imports mixed with type imports.

```ts
import React, { type FC, type ReactNode } from 'react';
```

should be 
```ts
import type { FC, ReactNode } from 'react';
import React from 'react';
```


To do so `fixStyle` should be `separate-type-imports`

See: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-imports.md#fixstyle

Reply:
Yeah we did discuss this and personally I agree with @ash-vd : https://github.com/kadena-community/kadena.js/pull/846#discussion_r1311427795

What do you think?

Reply:
I'm in favour of separating imports between "types" and "regulars".
It adds cognitive load when reviewing. You'd have to keep track of two things in one line; types and non-types
It's easier to scan all non-types when they aren't mixed
Especially when re-exporting certain types that are for the consumer. In index.ts files this happens often, when working with @kadena/client for example